content/src/denylist/DenylistServiceDecorator.ts

```diff
@@ -255,6 +255,10 @@ export class DenylistServiceDecorator implements MetaverseContentService {
     })
   }
 
+  getDeploymentsByHash(hash: string) {
+    return this.service.getDeploymentsByHash(hash)
```

Comment:
Should we remove from this list the denied items?

Reply:
We discussed with @nchamo that it didn't make too much sense to filter anything, since it's for troubleshooting and having an entity id won't let you have the deployment